ci/azure-pipelines-dev.yml

```diff
@@ -184,14 +184,17 @@ stages:
           name: CheckRepoRootSize
 
     - job: docs_snippet_checker
-      condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.DocsChanged'], true)
+      # condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.DocsChanged'], true)
       steps:
-      - task: PythonScript@0
-        inputs:
-          scriptSource: 'filePath'
-          scriptPath: 'ci/checks/validate_docs_snippets.py'
-          failOnStderr: true
-        name: DocsSnippetChecker
+        - script: |
+            yarn install
+          name: YarnInstall
+        - task: PythonScript@0
+          inputs:
+            scriptSource: 'filePath'
+            scriptPath: 'ci/checks/validate_docs_snippets.py'
+            failOnStderr: false
+          name: DocsSnippetChecker
```

Comment:
Could we make sure this works with a test change before merging? 🙇🏽

Reply:
Yep! I uncommented so the pipeline would run: https://github.com/great-expectations/great_expectations/pull/8217/checks?check_run_id=14666048161

<img width="1014" alt="Screenshot 2023-06-29 at 12 49 04 PM" src="https://github.com/great-expectations/great_expectations/assets/2117313/5e9da131-c02a-4003-9ea9-7738b45cc91b">

Passing now